src/components/Reusable/index.js

```diff
@@ -0,0 +1,19 @@
+import Text from './Text'
+import Button from './Button'
+import Card from './Card'
+import Header from './Header'
+import Icon from './Icon'
+import Image from './Image'
+import Main from './Main'
+import NavBar from './NavBar'
+
+export {
+    Text,
+    Button,
+    Card,
+    Header,
+    Icon,
+    Image,
+    Main,
+    NavBar,
+}
```

Comment:
suggestion instead writing this way, you can make a inde.js and write export {your component} from './components
if you are using default way then write {default as component}

Reply:
made the changes. thanks.